pkg/server/direct_protected_entity_manager.go

```diff
@@ -43,6 +44,10 @@ func NewDirectProtectedEntityManager(petms []astrolabe.ProtectedEntityTypeManage
 	}
 	for _, curPETM := range petms {
 		returnPEM.typeManager[curPETM.GetTypeName()] = curPETM
+		switch curPETM.(type) {
+		case *pvc.PVCProtectedEntityTypeManager:
+			curPETM.(*pvc.PVCProtectedEntityTypeManager).SetProtectedEntityManager(returnPEM)
```

Comment:
i didn't entirely understand why SetProtectedEntityManager is being done, could you explain this.

Reply:
Needs it so it can use the parent PEM to find the component PEs.  Doing it at object initialization time wound up being a funny loop.  This will need to be cleaned up and generalized as more second level PEs get introduced.